kategory-docs/docs/_data/menu.yml

```diff
@@ -2,84 +2,127 @@ options:
   - title: Quick Start
     url: /docs/index
 
+  - title: Blogs & Presentations
+    url: /docs/blogs
+
+  - title: Projects & Examples
+    url: /docs/projects
+
+  - title: Patterns
+
+    nested_options:
+
+      - title: Glossary
+        url: /docs/patterns/glossary/
+
+      - title: Error Handling
+        url: /docs/patterns/error_handling/
+
+      - title: Monad Comprehensions
+        url: /docs/patterns/monad_comprehensions/
+
+      - title: Comonad Comprehensions
+        url: /docs/patterns/comonad_comprehensions/
+
+      - title: Applicative Builder
+        url: /docs/patterns/applicative_builder/
+
+      - title: Polymorphic programs
+        url: /docs/patterns/polymorphic_programs/
+
+      - title: Free Algebras
+        url: /docs/patterns/free_algebras/
+
+      - title: Dependency Injection
+        url: /docs/patterns/dependency_injection/
+
+      - title: Monad Transformers
+        url: /docs/patterns/monad_transformers/
+
+      - title: Concurrency
+        url: /docs/patterns/concurrency/
+
+      - title: Optics
+        url: /docs/patterns/optics/
+
   - title: Data Types
 
     nested_options:
 
-     - title: Option
-       url: /docs/datatypes/option/
+      - title: Option
+        url: /docs/datatypes/option/
 
-     - title: OptionT
-       url: /docs/datatypes/optiont/
+      - title: Either
```

Comment:
I've reordered some menu items to favor those most commonly used first, and moving the transformers to the bottom.